File: app/src/main/java/cc/smartcash/wallet/Services/SmartAPIService.java
package cc.smartcash.wallet.Services;


import cc.smartcash.wallet.Models.SmartApiDefaultPrice;
import retrofit2.Call;
import retrofit2.http.GET;

public interface SmartAPIService {

    @GET("exchange/currencies")
    Call<SmartApiDefaultPrice> GetDefaultPrices();
}

File: app/src/main/java/cc/smartcash/smarthub/services/SmartAPIService.kt
package cc.smartcash.smarthub.services


import cc.smartcash.smarthub.models.SmartApiDefaultPrice
import retrofit2.Call
import retrofit2.http.GET

interface SmartAPIService {

    @GET("exchange/currencies")
    fun getDefaultPrices(): Call<SmartApiDefaultPrice>
}